src/utils/upload/processUpload.js

```diff
@@ -18,7 +18,7 @@ import endUserMessages from 'utils/endUserMessages';
 import pushNotificationMessage from 'utils/pushNotificationMessage';
 
 const prepareAndUploadFileToS3 = async (
-  file, uploadUrlParams, type, onStatusUpdate = () => { },
+  file, uploadUrlParams, type, onStatusUpdate = () => { }, abortController,
```

Comment:
I think adding the `abortController = null` default value makes more sense here than inside `processMultipartUpload(`. I would choose one of the two functions to define the default values to make it more readable.